src/main/java/net/atos/zac/app/klanten/KlantenUtil.java

```diff
@@ -0,0 +1,30 @@
+package net.atos.zac.app.klanten;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+
+public final class KlantenUtil {
+    public static final String ONBEKEND = "<onbekend>";
+
+    private static final char NBSP = '\u00A0'; // non breaking space
+
+    private KlantenUtil() {}
+
+    public static String hard(final String part) {
```

Comment:
Meer beschrijvende naam gebruiken? Bijvoorbeeld: replaceSpaceWithNonBreakingSpace?

En moet deze methode dan niet aangeroepen voor elk part in de hard(final String... parts) methode?